packages/server/__snapshots__/3_config_spec.js

```diff
@@ -92,7 +92,7 @@ exports['e2e config applies defaultCommandTimeout globally 1'] = `
 
   1) short defaultCommandTimeout
        times out looking for a missing element:
-     AssertionError: Timed out retrying: Expected to find element: \`#bar\`, but never found it.
+     AssertionError: Timed out retrying after  XX:XX: Expected to find element: \`#bar\`, but never found it.
```

Comment:
why's this one double `after  XX:XX`? 

I think this is matching one of our e2e helpers that was meant to replace the duration of table printouts, so in this context - it doesn't really make sense to replace here since the number shouldn't be a variable number (should always be the static configuration timeout). https://github.com/cypress-io/cypress/blob/develop/packages/server/test/support/helpers/e2e.ts#L158:L158

You can probably expand the RE matcher here to exclude these `after xxxms` messages. https://github.com/cypress-io/cypress/blob/develop/packages/server/test/support/helpers/e2e.ts#L47:L47